artifact/static/artifact/js/controllers/location_controller.js

```diff
@@ -1,133 +1,382 @@
-(function(){
-
+(function() {
 	//Angular App Module and Controller
-	angular.module('app').controller('LocationController', ['$scope', '$http', '$timeout', 'djangoUrl', function ($scope, $http, $timeout, $djangoUrl) {
-		var mc = this;
-		var GET_LOC = $djangoUrl.reverse('artifact:map_location', [$scope.map_id]);
-		var GET_MARKERS= $djangoUrl.reverse('artifact:markers', [$scope.map_id]);
-		var GET_CSV= $djangoUrl.reverse('artifact:csv_points', [$scope.map_id]);
-		var responsePromise = $http.get(GET_LOC);
-		responsePromise.success(function(data, status, headers, config) {
-			var mapOptions = {
-				zoom: data.zoom,
-	      		center: new google.maps.LatLng(data.latitude, data.longitude)
+	angular.module('app')
+		.controller('LocationController', ['$scope', '$http', '$timeout',
+            'djangoUrl',
+            function($scope, $http, $timeout, $djangoUrl)
+			{
```

Comment:
the indentation in this file is off, ideally for it would be 2 or 4 space indentation.